adagrams/game.py

```diff
@@ -1,11 +1,86 @@
-def draw_letters():
-    pass
+import random
+from typing import Counter
+letter_bank = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
 
+def draw_letters():
+    
+    letter_pool = [key for key, val in letter_bank.items() for i in range(val)]
+    last_index = len(letter_pool) - 1
+    hand = []
+    
+    while last_index > 0:
+        rand_index = random.randint(0, last_index)
+        letter_pool[last_index], letter_pool[rand_index] = letter_pool[rand_index], letter_pool[last_index]
```

Comment:
This is a pretty cool approach! It took me minute to figure out the intent here, but it seems like this is just creating a list with all the letters times their frequency values and then effectively "shuffling" them. This might be a good opportunity to add a comment about the intent of what the code is doing here.